### PR TITLE
Add r-pim using the conda-forge-r-helper script

### PR DESCRIPTION
**Pull request by pettyalex**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by pettyalex** [comment]
@conda-forge/help-r I believe this is ready to go

File: recipes/r-pim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    - r-nleqslv
+
+test:
+  commands:
+    - $R -e "library('pim')"           # [not win]
+    - "\"%R%\" -e \"library('pim')\""  # [win]
+
+about:
+  home: https://github.com/CenterForStatistics-UGent/pim
+  license: GPL-2.0-only
```
Comment (mfansler) [inline recipes/r-pim/meta.yaml:43]: DESCRIPTION shows `GPL (>= 2)`, so...

```suggestion
  license: GPL-2.0-or-later
```

File: recipes/r-pim/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  summary: 'Fit a probabilistic index model as described in Thas et al, 2012: <doi:10.1111/j.1467-9868.2011.01020.x>.
+    The interface to the modeling function has changed in this new version. The old
+    version is still available at R-Forge.'
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```
Comment (mfansler) [inline recipes/r-pim/meta.yaml:49]: With `GPL (>= 2)` I usually include both, which is consistent with how CRAN website displays it.

```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for submitting this! Just a minor touchup on the license and should be good.

**Review by mfansler — APPROVED** [review]

**Pull request merged**